src/Commands/ContentpoolClientCommands.php

```diff
@@ -27,19 +27,34 @@ class ContentpoolClientCommands extends DrushCommands {
   use RemotePullManagerTrait;
   use ReplicationHelperTrait;
 
+  /**
+   * Checks the remote for the content changes and does required pulls.
+   *
+   * @usage contentpool-client:check
+   *   drush cpc
+   *
+   * @command contentpool-client:check
+   * @aliases cpc
+   */
+  public function checkContent() {
+    $check_count = $this->getRemotePullManager()->checkAndDoAutopulls();
+
+    $this->output()->writeln("Checked and pulled {$check_count} remotes");
+  }
+
   /**
    * Pulls content from the contentpool server.
    *
    * @usage contentpool-client:pull-content
-   *   drush cpc
+   *   drush cppull
    *
    * @command contentpool-client:pull-content
-   * @aliases cpc
+   * @aliases cppull
    */
   public function pullContent() {
     $pull_count = $this->getRemotePullManager()->pullAllRemotes();
 
-    $this->output()->writeln("Scheduled pulling from {$pull_count} remotes");
```

Comment:
hm, is there this "check" necessary as well? not sure what check means. it needs to create a queue item and run the pull here.